### DISCUSSION TITLE
The problem of pymatgen package in OFM machine learning

### DISCUSSION THREAD

**Discussion by tuantuan-lin** [arosen93/QMOF#16] [Q&A] [answered]
This is my prolem,ImportError: cannot import name 'VoronoiCoordFinder' from 'pymatgen.analysis.structure_analyzer'.
What is the version of pymatgen package?Has the function been removed? How can I solve this problem?
Thank you very much.

**Comment by Andrew-S-Rosen** [comment, marked as answer]
Hi @tuantuan-lin. I moved this to the Issues tab. Let's continue the discussion here: https://github.com/arosen93/QMOF/issues/17.
